README.md

```diff
@@ -358,7 +358,7 @@ example,
 cd ~
 
 # Download the library files
-wget "https://github.com/microsoft/onnxruntime/releases/download/v1.14.1/onnxruntime-linux-x64-1.14.1.tgz"
+wget "http://github.com/microsoft/onnxruntime/releases/download/v1.14.1/onnxruntime-linux-x64-1.14.1.tgz"
```

Comment:
Why change this to HTTP? GitHub uses HSTS, so this gets redirected to HTTPS anyway

![image](https://github.com/usnistgov/ns3-oran/assets/54951512/0a3ca768-e4e0-4318-a961-aee62776c86f)

Reply:
It is because the certificate for HTTPs fails so the download does not complete, so instead of using wget and the --no-check-certificate, suggesting to use HTTP.

Reply:
Where are you getting the certificate failure? I can't replicate on macOS or Linux

Reply:
It's only with torch so we can use https for onnx. I was just suggesting using http to be consistent but also so that cert issues aren't a problem later.